tests/conftest.py

```diff
@@ -13,12 +13,27 @@
     "site_config_dir",
 )
 
+PROPS_PATH = (
+    "user_data_path",
+    "user_config_path",
+    "user_cache_path",
+    "user_state_path",
+    "user_log_path",
+    "site_data_path",
+    "site_config_path",
+)
+
 
 @pytest.fixture(params=PROPS)
 def func(request: SubRequest) -> str:
     return cast(str, request.param)
 
 
+@pytest.fixture(params=PROPS_PATH)
+def func_path(request: SubRequest) -> str:
+    return cast(str, request.param)
+
+
 @pytest.fixture()
```

Comment:
We should have a test somewhere that ensures that for every dir variant we have a path variant. This is true for both methods and the class properties. We should also assert that their interface otherwise is in sync. To ensure we don't drift the two variants apart.

Reply:
This still needs addressed 👍